parcelgen-runtime/src/com/yelp/parcelgen/JsonUtil.java

```diff
@@ -49,6 +49,66 @@ private JsonUtil() {
         return list;
     }
 
+    public static List<Boolean> parseBooleanJsonList(JSONArray array)
+            throws JSONException {
+        if (array == null) {
```

Comment:
if the input is `null` (distinct from empty list), i think we should return `null`